message.go

```diff
@@ -99,110 +99,95 @@ func (c *Client) DeleteMessage(ctx context.Context, messageIDs ...string) error
 func (c *Client) MarkMessagesRead(ctx context.Context, messageIDs ...string) error {
 	pages := xslices.Chunk(messageIDs, maxPageSize)
 
-	return parallel.DoContext(ctx, runtime.NumCPU(), len(pages), func(ctx context.Context, idx int) error {
-		defer async.HandlePanic(c.m.panicHandler)
+	for _, page := range pages {
```

Comment:
James preferred this :wink: 

```suggestion
	for _, page := range xslices.Chunk(messageIDs, maxPageSize) {
```

Reply:
You guys are not fan of debuggers :D

Reply:
Done